src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -44,8 +43,7 @@ const InputPassword: UISchemaElementComponent<{
   return (
     <Box>
       <PasswordInput
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...(!hideLabel && { label })}
```

Comment:
Trying to hide the label was causing issues. it was only used for security question verification but it was using a paragraph tag as the label. This doesnt match parity and did not pass a11y checks. We need to use the built in label with this component.

Reply:
which a11y check it fails?

Reply:
btw, it's allowed to hide label for input
https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text

Reply:
If we were using MUI's raw component, absolutely (there is a dedicated attr for that) but this is ODY's component we are using and it does not allow it. You can even see in the snapshot there was still a label attached to the DOM with no value with the `for` attached to the credentials.answer field. 

https://github.com/okta/odyssey/blob/develop/packages/odyssey-react-mui/src/components/PasswordInput/PasswordInput.tsx

Reply:
we will want the form make sense for the end user first, it's not about which comp library we are using, if there is an issue, then we should report and fix it

Reply:
We should also ask Odyssey to make the label more flexible, but we can stick with the question as the label for now since it meets parity and doesn't seem to violate a11y requirements.